src/Sulu/Component/Content/Document/Subscriber/RedirectTypeSubscriber.php

```diff
@@ -28,7 +28,7 @@ public static function getSubscribedEvents()
     {
         return [
             Events::METADATA_LOAD => 'handleMetadataLoad',
-            Events::PERSIST => 'handlePersist',
+            Events::PERSIST => ['handlePersist', 15],
```

Comment:
Yeah, would be really nice to be able to define the dependencies among these subscribers instead of struggling with priorities. That would probably be the better solution for even more places we are using priorities. But that's definitely out of scope of this PR.